seed/data/referral/referral.js

```diff
@@ -1,4 +1,4 @@
-export default [
```

Comment:
This file probably shouldn't be here since it's just a temporary file we're using until we hook into the Drupal data via the API. If we leave it here, it will be persisted to MongoDB when the seeder runs.

Reply:
done